src/TaskRunner/Commands/CloneCommands.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace EcEuropa\Toolkit\TaskRunner\Commands;
+
+use OpenEuropa\TaskRunner\Commands\AbstractCommands;
+use OpenEuropa\TaskRunner\Tasks as TaskRunnerTasks;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Provides commands to clone a site for development and a production artifact.
+ */
+class CloneCommands extends AbstractCommands {
+
+  use TaskRunnerTasks\CollectionFactory\loadTasks;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getConfigurationFile() {
+    return __DIR__ . '/../../../config/commands/clone.yml';
+  }
+
+  /**
+   * Install clone from production snapshot.
+   *
+   * It restores the database and imports the configuration.
+   * - Verify if the dumpfile exists.
+   * - Import configuration from sync into active storage.
+   * - Execute cache-rebuild.
+   *
+   * @param array $options
+   *   Command options.
+   *
+   * @return \Robo\Collection\CollectionBuilder
+   *   Collection builder.
+   *
+   * @command toolkit:install-dump
+   *
+   * @option uri Drupal uri.
+   * @option dumpfile Drupal uri.
+   */
+  public function installDump(array $options = [
+    'uri' => InputOption::VALUE_REQUIRED,
+    'dumpfile' => InputOption::VALUE_REQUIRED,
+  ]) {
+    $tasks = [];
+
+    if (!file_exists($options['dumpfile'])) {
+      $this->say('"' . $options['dumpfile'] . '" file not found, use the command "toolkit:download-dump --dumpfile ' . $options['dumpfile'] . '".');
+
+      return $this->collectionBuilder()->addTaskList($tasks);
+    }
+
+    // Unzip and dump database file.
+    $tasks[] = $this->taskExecStack()
+      ->stopOnFail()
+      ->exec('vendor/bin/drush --uri=' . $options['uri'] . ' sql-drop -y')
+      ->exec('vendor/bin/drush --uri=' . $options['uri'] . ' sqlc < ' . $options['dumpfile'])
+      ->exec('vendor/bin/drush --uri=' . $options['uri'] . ' updatedb -y')
+      ->exec('vendor/bin/drush --uri=' . $options['uri'] . ' cache:rebuild')
+      ->exec('vendor/bin/drush --uri=' . $options['uri'] . ' config:import -y')
+      ->exec('vendor/bin/drush --uri=' . $options['uri'] . ' cache:rebuild');
+
+    // Build and return task collection.
+    return $this->collectionBuilder()->addTaskList($tasks);
+  }
+
+  /**
+   * Download production snapshot.
+   *
+   * In order to make use of this functionality you must add your
+   * ASDA credentials to your environment like. If the credentials
+   * are not there you will be prompted to insert them.
```

Comment:
Important refer that is a sanitized dump and is done automatically during the nigth, not done from production on demand.

Also maybe we can use `Download ASDA snapshot.`

Reply:
Done.